docs/section23/Rule23-9.md

```diff
@@ -0,0 +1,51 @@
+
+# Airside System - Rule 23-9 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** True  
+**Rule ID:** 23-9  
+**Rule Description:** System 11 Minimum volume setpoint shall be the largest of 50% of the maximum design airflow rate, the minimum ventilation airflow rate or the airflow required to comply with codes or accredidation standards.    
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** System 11 Minimum volume setpoint shall be the largest of 50% of the maximum design airflow rate, the minimum ventilation airflow rate or the airflow required to comply with codes or accredidation standards.  
+**Data Lookup:** None  
+**Evaluation Context:** HVAC System  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-11.  
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. baseline_system_type_compare()
+
+**Applicability Checks:**  
+- create a list of the target system types: `APPLICABLE_SYS_TYPES = [HVAC_SYS.SYS_11]`
+- Get B-RMR system types: `baseline_system_types_dict = get_baseline_system_types(B-RMR)`
+
+- loop through the applicable system types: `for target_sys_type in APPLICABLE_SYS_TYPES:`
+    - and loop through the baseline_system_types_dict to check the system types of the baseline systems: `for system_type in baseline_system_types_dict:`
+        - do baseline_system_type_compare to determine whether the baseline_system_type is the applicable_system_type: `if((baseline_system_type_compare(system_type, target_sys_type, false)):`
+            - the systems in the list: baseline_system_types_dict[system_type] are sys-11 - loop through the list of systems: `for hvac_system_id in baseline_system_types_dict[system_type]:`
+            - `CONTINUE TO RULE LOGIC`
+            - otherwise, rule not applicable: `else: RULE_NOT_APPLICABLE`
+ 
+**Rule Logic**
+    - get the hvac system: `hvac_system = get_object_by_id(hvac_system_id)`
+    - get the fan system: `fan_system = hvac_system.fan_system`
+    - get the fan system minimum volume flow rate: `min_volume_flowrate = fan_system.minimum_airflow`
+    - get the fan system minimum ventilation air flow rate: `min_ventilation_flowrate = fan_system.minimum_outdoor_airflow`
+    - create a variable for the maximum volume flow rate: `maximum_supply_flowrate = 0`
```

Comment:
Isn't this rule something that should be checked at the terminal level? See [19-7](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section19/Rule19-7.md) where we do OA related checks at the terminal level. I think it is more in-line with modeling to check each terminal to see if minimum_airflow/primary_airflow because this is where the min flow ratio is typically set. @JacksonJ-KC what do others think? We could  then report any terminal units that violate the rule in the UNDETERMINED or FAIL message.